taskcluster/rb_taskgraph/build.py

```diff
@@ -42,3 +44,12 @@ def add_variant_config(config, tasks):
                 apks[apk["abi"]] = apk_name
 
         yield task
+
+
+@transforms.add
+def populate_version_name(config, tasks):
+    for task in tasks:
+        if task.getdefault("attributes", {})["nightly"]:
```

Comment:
Is `getdefault` a taskgraph thing? or did you mean `task.get("attributes", {}).get("nightly")` ?

Reply:
`[task 2019-08-20T00:13:58.463Z] AttributeError: 'dict' object has no attribute 'getdefault'` :)

Reply:
Yeah, `getdefault` wasn't correct. I grafted Tom's approach from #878